Support user access token authentication (#6)
rune-labs/runeq-python: docs/pages/quickstart.rst

    from runeq import Config

    cfg = Config('~/.rune/config.yaml')

In the example above, configuration was loaded from a `YAML <https://yaml.org/>`_-formatted
file. See the `example config <https://github.com/rune-labs/runeq-python/blob/master/example_config.yaml>`_
for the expected contents of this file.

> If you are doing multi patient analysis it is recommended to use an access token
> as this method of authentication will give you access to all patients in your org.

Reviewer: A little more styling, because this is user-facing documentation:

```suggestion
If you are doing multi-patient analysis, it is recommended to use an **access token** to
authenticate, as it will provide access to all the patients in your organization.
```

.. code-block:: python

    accel = v1client.Accel(

>         patient_id='992967a09cad48378f7b628aff5bdf6c',

Reviewer: good call!
